packages/devel/ncurses/package.mk

```diff
@@ -17,7 +17,8 @@
 ################################################################################
 
 PKG_NAME="ncurses"
-PKG_VERSION="6.0-20170812"
+PKG_VERSION="6.0-20171007"
+PKG_SHA256="d74ce1f651824b6254a6d913d312e4ab32874812bb066288ef87b0b0482c4db5"
 PKG_REV="1"
```

Comment:
this `PKG_REV="1"` can also be removed (and the one in `packages/security/openssl/package.mk`) if you want to do some more cleanup :)

Reply:
Done. I also removed PKG_REV from `tini` and `libnetwork`.